eelbrain/_experiment/mne_experiment.py

```diff
@@ -6727,12 +6727,12 @@ def _set_analysis_options(
 
         # pmin
         if pmin is not None:
-            # source connectivity
-            connectivity = self.get('connectivity')
-            if connectivity and not data.source:
-                raise NotImplementedError(f"connectivity={connectivity!r} is not implemented for data={data!r}")
-            elif connectivity:
-                items.append(connectivity)
+            # source adjacency
+            adjacency = self.get('adjacency')
+            if adjacency and not data.source:
+                raise NotImplementedError(f"adjacency={adjacency!r} is not implemented for data={data!r}")
```

Comment:
```suggestion
                raise NotImplementedError(f"{adjacency=} is not implemented for {data=}")
```
I'm updating those whenever I come across them